build.sbt

```diff
@@ -61,6 +61,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.slick"           %% "slick"                         % slickVersion exclude ("org.reactivestreams", "reactive-streams") exclude ("com.typesafe", "config") exclude ("org.slf4j", "slf4j-api"),
   "com.typesafe.slick"           %% "slick-hikaricp"                % slickVersion exclude ("org.slf4j", "slf4j-api"),
   "com.typesafe.slick"           %% "slick-codegen"                 % slickVersion,
+  "com.github.tminglei"          %% "slick-pg"                     % "0.18.0",
```

Comment:
I didn't highlight this at first, but this is a rather strong choice that will restrict the database to be postgres, right?
I don't remember if Slick is able to fallback to some alternative solution if the actual database is not postgres and we use one of the custom apis from this plugin-library.

This would work fine as long as we stick to using Postgres (which is not necessarily forever, if we look at what is said in the main project readme)

This is not a blocking change for me, but I want to raise a flag for global awareness.

Reply:
There are many things that will need to be changed if we stop using postgres, but yes this will be one of them.